src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
+// Working with data
+// Making array from textContant
 
-// write your code here
+const populationData = Array.from(document.querySelectorAll('span.population'));
+
+// Convert data to String
+const dataToString = populationData.map((span) => span.textContent);
+
+// Convert data to Number
+const dataToNums = dataToString.map((str) => +str.replaceAll(',', ''));
+
+// Receiving of sum from all numbers
+const totalPopulation = dataToNums.reduce((a, b) => a + b);
```

Comment:
Consider providing an initial value for the `reduce` function to handle cases where `dataToNums` might be empty. This will prevent potential errors.